_dashboards/get-started/time-filter.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title: Set the time range
+parent: OpenSearch Dashboards core concepts
+nav_order: 10
+---
+
+## Set the time range
+
+You can change the time range to display dashboard data over minutes, hours, days, weeks, months, or years.
+
+The default time range is **Last 15 minutes**. You can change the time range at the dashboard level or 
+{: .note}
+
+To change the time range at the dashboard level:
+
+1. Select the calendar icon.
+1. Choose one of the following:
+   - **Quick select:** Choose a time based on the last or next number of seconds, minutes, hours, days, or other time unit. 
+   - **Commonly used:** Choose a common time range like **Today**, **Last 7 days**, and **Last 30 days**. 
+   - **Recently used date ranges:** Select a previously used time range.
+   - **Refresh every:** Set an automatic refresh period.
+  
+![Time range interface]({{site.url}}{{site.baseurl}}/images/time-range.png)
+
+1. Select **Show dates** to set start and end times.
```

Comment:
Here it's not enough to just select Show dates. The user then needs to click inside the date range to bring up the tooltip where Absolute, Relative, and Now are. I would add it to step 1.

Reply:
Change made.